serverless.yaml

```diff
@@ -18,6 +18,10 @@ custom:
   region: ${opt:region, self:provider.region}
   oauthRedirect: ${opt:oauthRedirect, 'http://localhost'}
   config: ${file(serverless_config.json)}
+  bundle:

```

Comment:
In the options https://github.com/AnomalyInnovations/serverless-bundle#options it looks like we may want to specify `packager: yarn` thoughts?

Reply:
yep, good idea. I'll change it to that